tests/test_board.py

```diff
@@ -1,70 +1,113 @@
 from engine.board import Board
 from engine.cardstack import CardStack
+
 import pytest
-import unittest
+import util
+
+
+@pytest.fixture(scope="module")
+def card():
+    return util.create_mock_card(number=2, color='blue')
+
+
+@pytest.fixture(scope="module")
+def your_card():
+    return util.create_mock_yourcard(public_number=False, public_color=False)
+
+
+@pytest.fixture(scope='module')
```

Comment:
Should probably make this scope "function" level? What if an early test relies on drawing cards and a later test assumes a fresh deck for a new board?

More broadly, if you can find a writeup about `scope='function' vs scope='module'` idiomatic testing, that'd be wonderful. I'd like to read more about this but can't find anything.

Reply:
The `mock_deck` shouldn't have any cards in it. If a `Board` test requires that functionality then a new/different fixture should be created to support that test. A value or generator passed as `return_draw_card` to `util.create_mock_deck` does allow that.

As far as idiomatic scoping goes I didn't find anything either. The `py.test` documentation generally defines **all** its fixtures in a single `conftest.py`.